<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEventDialog</class>
 <widget class="QDialog" name="AddEventDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aggiungi un evento</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="NameEventLabel">
          <property name="text">
           <string>Nome dell'evento:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="NameLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="DateLabel">
          <property name="text">
           <string>Data:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="DateEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="AllDayCheckBox">
        <property name="text">
         <string>Tutto il giorno</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="StartTimeLabel">
          <property name="text">
           <string>Orario di inizio:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="StartTimeEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="EndTimeLable">
          <property name="text">
           <string>Orario di fine:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="EndTimeEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="NoteLabel">
      <property name="text">
       <string>Note:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="NoteEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="AddButton">
      <property name="text">
       <string>Aggiungi evento</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
